src/helpers.js

```diff
@@ -468,25 +468,32 @@ function addSelectors(element, selectors) {
     });
   }
 
-  function findElement(element, selector, args) {
-    if (selector) {
-      var replacedSelector = selector.replace(/{(\d+)}/g, function (match, group) {
-        return args[group];
-      });
-      return element.find(replacedSelector);
+  function findElement(element, value, args) {
+    if (value) {
+      if (angular.isString(value) || value.selector) {
+        var replacedSelector = (value.selector || value).replace(/{(\d+)}/g, function (match, group) {
+          return args[group];
+        });
+        return element.find(replacedSelector);
+      } else if (value.repeater) {
+        var elements = element.find(value.repeater);
+        return angular.isDefined(args[0]) ? elements.eq(args[0]) : elements;
+      }
     }
     return angular.element(element);
   }
 
   angular.forEach(selectors, function (value, key) {
-    var selector = angular.isObject(value) ? value.selector : value,
-      sub = value.sub,
-      children = value.children;
+    var
```

Comment:
Deze refactor snap ik niet helemaal. `value` hoeft toch niet per se `children`/`sub` te hebben? `value` kan ook de selector zelf zijn (dat is de simpelste use case)